sway-lsp/src/traverse/lexed_tree.rs

```diff
@@ -344,18 +344,23 @@ impl Parse for ItemTrait {
 
 impl Parse for ItemImpl {
     fn parse(&self, ctx: &ParseContext) {
+        dbg!();
```

Comment:
Spurious `dbg!()`, repeated 5 times in this function.

Reply:
Done.